card/spec/lib/card/stage_director_spec.rb

```diff
@@ -219,7 +219,7 @@
   describe 'subcards' do
     it "has correct name if supercard's name get changed" do
       Card::Auth.as_bot do
-        $changed = false
+        changed = false
```

Comment:
Is used in a completely different context (in a event on a card object but with the rspec binding as fallback for missing methods). I wrote the test to fix a bug and I wanted to make sure that it doesn't fail because of this variable.